nixos/tests/rxe.nix

```diff
@@ -26,27 +26,27 @@ in {
 
   testScript = ''
     # Test if rxe interface comes up
-    $server->waitForUnit("default.target");
-    $server->succeed("systemctl status rxe.service");
-    $server->succeed("ibv_devices | grep rxe0");
+    server.wait_for_unit("default.target")
+    server.succeed("systemctl status rxe.service")
+    server.succeed("ibv_devices | grep rxe0")
 
-    $client->waitForUnit("default.target");
+    client.wait_for_unit("default.target")
 
     # ping pong test
-    $server->succeed("screen -dmS rc_pingpong ibv_rc_pingpong -p 4800 -g0");
-    $client->succeed("sleep 2; ibv_rc_pingpong -p 4800 -g0 server");
+    server.succeed("screen -dmS rc_pingpong ibv_rc_pingpong -p 4800 -g0")
+    client.succeed("sleep 2; ibv_rc_pingpong -p 4800 -g0 server")
 
-    $server->succeed("screen -dmS uc_pingpong ibv_uc_pingpong -p 4800 -g0");
-    $client->succeed("sleep 2; ibv_uc_pingpong -p 4800 -g0 server");
+    server.succeed("screen -dmS uc_pingpong ibv_uc_pingpong -p 4800 -g0")
+    client.succeed("sleep 2; ibv_uc_pingpong -p 4800 -g0 server")
 
-    $server->succeed("screen -dmS ud_pingpong ibv_ud_pingpong -p 4800 -s 1024 -g0");
-    $client->succeed("sleep 2; ibv_ud_pingpong -p 4800 -s 1024 -g0 server");
+    server.succeed("screen -dmS ud_pingpong ibv_ud_pingpong -p 4800 -s 1024 -g0")
+    client.succeed("sleep 2; ibv_ud_pingpong -p 4800 -s 1024 -g0 server")
 
-    $server->succeed("screen -dmS srq_pingpong ibv_srq_pingpong -p 4800 -g0");
-    $client->succeed("sleep 2; ibv_srq_pingpong -p 4800 -g0 server");
+    server.succeed("screen -dmS srq_pingpong ibv_srq_pingpong -p 4800 -g0")
+    client.succeed("sleep 2; ibv_srq_pingpong -p 4800 -g0 server")
```

Comment:
This looks like it could be:

```python
for str in "rc", "uc", "ud", "srq":
    server.succeed("screen -dmS {}_pingpong ibv_{}_pingpong -p 4800 -g0".format(src, src))
    client.succeed("sleep 2; ibv_{}_pingpong -p 4800 -s 1024 -g0 server".format(src))
```

Reply:
Yes, that is shorter.